style.css

```diff
@@ -0,0 +1,183 @@
+.nav {
+  display: grid;
+  grid-template-columns: 1fr 2fr 1fr;
+  grid-template-rows: minmax(100px, auto);
+  align-items: center;
+}
+
+.nav .logo {
+  height: 36px;
+}
+
+.nav > div {
+  padding: 1em;
+  /* background-color: #666; */
+}
+
+/* searchbar */
+.nav * {
+  box-sizing: border-box;
+}
+
+/* Style the search field */
+.nav form.searchbar input[type="text"] {
+  padding: 10px;
+  font-size: 17px;
+  border: 1px solid grey;
+  float: left;
+  width: 80%;
+  background: #f1f1f1;
+}
+
+/* Style the submit button */
+.nav form.searchbar button {
+  float: left;
+  width: 20%;
+  padding: 10px;
+  background: #e4e4e4;
+  color: rgb(104, 104, 104);
+  font-size: 17px;
+  border: 1px solid grey;
+  border-left: none; /* Prevent double borders */
+  cursor: pointer;
+}
+
+.nav form.searchbar button:hover {
+  background: #cccaca;
+}
+
+/* Clear floats */
+.nav form.searchbar::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+/* Navigation Icons */
+.nav .icons {
+  justify-self: end;
+  color: #606060;
+}
+
+.nav .icons i {
+  padding-right: 16px;
+}
+
+/* Main Section */
+.main {
+  display: flex;
+  justify-content: space-between;
+}
+
+.main .video {
+  flex-basis: 75%;
+}
+.main .aside {
+  flex-basis: 23%;
+}
+.main .aside .item {
+  display: flex;
+}
+.main .aside .item div {
+  flex: 1;
+}
+.main .aside .item img {
+  width: 100%;
+  height: 85%;
+}
+
+.main .aside .item .vid_info {
+  display: flex;
+  /* flex: 1; */
+  flex-direction: column;
+  margin-left: 7px;
+}
+.main .aside .item .vid_info p {
+  margin: 0 0 2px 0;
+  padding: 0;
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+}
+
+.main .aside .item .vid_info p:first-child {
+  font-weight: bold;
+}
+
+.main .aside .item .vid_info .vid_auth {
+  font-size: 14px;
+  color: #9b9b9b;
+}
```

Comment:
[REQUIRED CHANGE] Please do not use too many selectors for styling an element. you are using unique classes. So, one or two classes are enough to style an element in most of the cases. [project requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png)
Please checkout this article. [reference](https://speckyboy.com/good-bad-css-practices/)